verification.py

```diff
@@ -2107,10 +2107,12 @@ def run_xi_beta_breakpoints(self):
                 f"n=100_alpha={alpha}",
                 sample_size=100,
                 Ne=Ne,
-                r=1e-7,
```

Comment:
Looks like something funny has happened here while you were rebasing. This is cancelling out some of the changes made in #1059. Probably easiest to inspect the code and manually revert your local changes.